build_tools/benchmarks/common/common_arguments.py

```diff
@@ -126,18 +133,18 @@ def check_exe_path(path):
   return parser
 
 
-def expand_and_check_file_paths(paths: Sequence[str]) -> List[str]:
+def expand_and_check_file_paths(paths: Sequence[str]) -> List[pathlib.Path]:
   """Expands the wildcards in the paths and check if they are files.
     Returns:
       List of expanded paths.
   """
 
   expanded_paths = []
   for path in paths:
-    expanded_paths += glob.glob(path)
+    expanded_paths += [pathlib.Path(path) for path in glob.glob(path)]
```

Comment:
pathlib also has globs: https://docs.python.org/3.7/library/pathlib.html#pathlib.Path.glob.

Reply:
`pathlib.Path("x").glob(pattern)` needs the patterns to be relative to the path "x". So it's not very suitable in this case as the path in the path list can be either relative or absolute.